docs/test.md

```diff
@@ -0,0 +1,34 @@
+# Test
+
+Most classes have unit tests (classname.spec.ts), though we also want to test the whole code in an "End-To-End" way.
+These tests are written in a separate test-folder, which can be found on the same level as src.
+
+To run tests, use the following command:
```

Comment:
Die Info, dass alle tests auf einmal ausgeführt werden wäre ganz gut (Unit und E2e

Reply:
geht das nicht aus dem kommentar über npm run test vor? (da steht ja: # to test all .spec files)